src/deck-options.md

```diff
@@ -302,7 +302,7 @@ Controls how Anki gathers cards from each subdeck. The options are:
   [reposition](https://docs.ankiweb.net/browsing.html#cards) cards in different
   ways.
 
-- Deck, then random notes: gathers cards from each deck in order, starting from the top.
+- Deck, then random notes: gather cards from each deck in order, starting from the top.
```

Comment:
I don't think it makes sense to change this without changing the others as well.